crypto/transport/encrypt.go

```diff
@@ -19,68 +19,65 @@ package transport
 import (
 	"crypto/aes"
 	"crypto/cipher"
-	"crypto/md5"
 	"crypto/rand"
 	"io"
-)
 
-func md5sum(d []byte) []byte {
-	h := md5.New()
-	h.Write(d)
-	return h.Sum(nil)
-}
+	"github.com/thunderdb/ThunderDB/crypto/hash"
+)
 
-func evpBytesToKey(password string, keyLen int) (key []byte) {
-	const md5Len = 16
+// KeyDerivation, according to ANSI X9.63 we should do a key derivation before using
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported function KeyDerivation should be of the form "KeyDerivation ..."